app/templates/sites/new.hbs

```diff
@@ -88,11 +87,30 @@
           }}
           @primaryContact={{@model.primaryContact}}
           @secondaryContact={{@model.secondaryContact}}
+          @errorMessage={{this.validationErrors}}
+          @warningMessage={{this.validationWarnings}}
         >
           <:title>Contactgegevens</:title>
 
         </ContactDataEditCard>
       </div>
     </form>
+    <AuModal @modalOpen={{this.showWarningModal}} @closable={{false}}>
+      <:title>There are warnings</:title>
+      <:body>Some fields in the form have warnings. Are you sure you want to
+        continue?</:body>
+      <:footer>
+        <AuButton
+          @skin="primary"
+          {{on "click" (perform this.handleWarningModalOK)}}
```

Comment:
Make a normal handler for this which closes the dialog and starts the save task so the user can see the loading animation